src/Layout/Schema/index.style.js

```diff
@@ -49,3 +49,77 @@ export const NextArrow = styled.img`
   width: 3rem;
   margin-left: 1.3rem;
 `;
+
+
+export const BodyPage = styled.html`
+  background: #cfcfcf;
+  justify-content: center;
+`;
+
+export const PopUPCard = styled.body`
+  text-align: center;
+  display: flex;
+  height: 0.6rem;
+`;
+
+export const PopUpBox = styled.div`
+  width: ${(props) => props.Width || '82rem'};
+  height: ${(props) => props.Height || '100rem'};
+  background: #fffcfc;
+  border: 0.1rem solid #c4c4c4;
+    margin-left: -17.4rem;
+    margin-top: -38rem;`;
+
+export const Logo = styled.div`
+    width: 82.5rem;
+    height: 34rem;
+    background: ${(props) => props.Background || '#205c8b'};
+    margin-top: -1rem;
+    margin-left: -0.4rem;
+`;
+
+export const LogoImage = styled.img`
+    width: 45rem;
+    height: 41rem;
+`;
+
+export const Container2 = styled.div`
+    display: flex;
+    flex-direction: column;
+`;
+
+export const PopUpDiv = styled.span`
+  padding: 4.4rem;
+  font-size: 3.5rem;
+  text-align: left;
+  letter-spacing: 0.1rem;
+`;
+
+export const ContainerBtn = styled.div`
+  display: flex;
+  justify-content: space-around;
+  margin-top: 15rem;
+`;
+
+
+export const YesBtn = styled.button`
```

Comment:
why you didn’t create one styled component and add these properties
you repeated them
```  border: 0.3rem #cecece solid;
    background-color: white;
    border-radius: 4rem;
    width: 16rem;
    height: 7rem;
    font-size: 3.5rem;
    outline: none;
```